cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -305,6 +308,7 @@ private BuildOptions constructBuildOptions() {
                 .cloud(cloud)
                 .dumpBir(dumpBIR)
                 .dumpBirFile(dumpBIRFile)
+                .nBal(nBal)
```

Comment:
don't think phrase `nBal` matches with rest of the java code, were we have use longer descriptive names.